README.md

```diff
@@ -1,7 +1,205 @@
 # recidiviz
 An in-progress effort to visualize recidivism metrics.
 
-## Data
+## Contents
+1. [Development](#development)
+1. [Tooling](#tooling)
+1. [Application Structure](#application-structure)
+1. [Data Sources](#data-sources)
+1. [License](#license)
+
+## Development
+
+### Getting set up
+1. Grab the source:
+
+   ```git clone git@github.com:Recidiviz/recidiviz.com.git```
+1. Install Yarn package manager:
+
+   ```brew install yarn```
+
+   For alternative Yarn installation options, see [Yarn Installation](https://yarnpkg.com/en/docs/install).
+1. Install dependencies:
+
+   ```yarn install```
+
+That's it! We suggest installing a linting package for your preferred code editor that hooks into [eslint](#eslint). We recommend [linter-eslint](https://atom.io/packages/linter-eslint) for Atom.
```

Comment:
Are there any concerns around eslint versioning, i.e. keeping the version in Atom consistent with the version installed in the app?

Reply:
No, this the precise reason for the approach I recommended. That Atom package looks for `eslint` (the Node module we're using for linting) and leverages it. This way, all we have to do is keep `eslint`+ plugins up to date and then linting in editor will be kept in lock-step.

Reply:
Nice!